SWIG/cashflows.i

```diff
@@ -607,6 +611,8 @@ Leg _FixedRateLeg(const Schedule& schedule,
                   const std::vector<Real>& nominals,
                   const std::vector<Rate>& couponRates,
                   BusinessDayConvention paymentAdjustment = Following,
+                  const Calendar& paymentCalendar = Calendar(),
+                  const Natural paymentLag = 0,
```

Comment:
I would put these last so they don't break existing calls. Same for the other similar constructors.

Reply:
Ok, I will put them last right away. But I am afraid that having `paymentCalendar = Calendar()` will break the existing calls anyway, giving `no calendar implementation provided` error. In the light of that, we could drop the `paymentCalendar` parameter for now, knowing that internally it will be defaulted from the `schedule`?

Reply:
I see — yes, it can break the call if it's not filtered.  I'm adding a suggestion below to do that.